templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <style>
+      #{{ snippet_id }}-ffos-circling-fox {
+          /* All CSS should be preceeded by the ID selector for this individual
+            snippet. */
+      }
+    </style>
+    
+    {% if icon_url %}<a href="#" id="hello-cta">{% endif %}
+    <img class="icon" src="{{ icon }}" />
+    {% if icon_url %}</a>{% endif %}
+    <p>{{ text|safe }}</p>
+
+    <script type="text/javascript">
+      //<![CDATA[
+      var TARGET_AVAILABLE;
+      var TARGET_NAME = 'loop';
```

Comment:
Having a huge list of constants at the start makes the code less readable than just putting the strings directly where they're used.